Bayesian_RCT.tex

```diff
@@ -84,51 +84,51 @@ \section{Bayesian Data Analysis}
 
 
 \subsection{Model estimation}
-Once the likeilihood model and prior distribution are specified, the posterior distribution of our outcome variable can be estimated. As discussed in section 2.1, the posterior distribution is obtained by simply multiplying the prior and likelihood functions. The distribution obtained by this process is proportional to the true posterior because we generally tend to ignore the normalizing constant (the denominator in equation 1). We use an approximation of the posterior as shown in equation 2 because calculating the true posterior analytically may be next to impossible.  Thus, a standard practice is to use Markov Chain Monte Carlo (MCMC) sampling methods to approximate the posterior up to a normalizing constant and sample from it. There are some other approaches to calculating the posterior distribution, but those are beyond the scope of this paper. We recommend using the Bayesian probabilistic modeling language Stan [SOURCES TO UNDERSTAND STAN]. Stan uses a Hamiltonian Monte Carlo sampling algorithm (from a broader class of MCMC sampling methods) to approximate the posterior distribution. Betancourt [2017] has a clear exposition of how the algorithm works. Stan returns to us the posterior distribution of the desired parameters. Stan can also predict values based on the specified model. These predictions are basically random draws from the posterior distribution and can be used for model checking, validation and expansion.
+Once the likeilihood model and prior distribution are specified, the posterior distribution of our outcome variable can be estimated. As discussed in section 2.1, the posterior distribution is obtained by simply multiplying the prior and likelihood functions. The distribution obtained by this process is proportional to the true posterior because we generally tend to ignore the normalizing constant (the denominator in equation 1). We use an approximation of the posterior as shown in equation 2 because calculating the true posterior analytically may be practically impossible.  A standard practice is to use Markov Chain Monte Carlo (MCMC) sampling methods to approximate the posterior up to a normalizing constant and sample from it. There are  other approaches to calculating the posterior distribution, but those are beyond the scope of this paper. Analyses in this paper are carried out with the Bayesian probabilistic modeling language Stan [SOURCES TO UNDERSTAND STAN]. Stan uses a Hamiltonian Monte Carlo sampling algorithm (from a broader class of MCMC sampling methods) to approximate the posterior distribution. Betancourt [2017] has a clear exposition of how the algorithm works. Stan returns to us the posterior distribution of the desired parameters. Stan can also predict values based on the specified model. These predictions are basically random draws from the posterior distribution and can be used for model checking, validation, and expansion.
```

Comment:
We need to justify why proportionality is sufficient to sample from. I think it just needs a citation or something like that.